### PR TITLE
created updates

### LINKED ISSUE #2: Preparing a common folder for deployment
Create two directories, each with a Dockerfile, and then a setup script

### PR DESCRIPTION
**Pull request by aleksandr-andreichenko-epam**

### CODE REVIEW

**Comment by aleksandr-andreichenko-epam** [comment]
created the new settings for saltstack automation

File: salt-minion/setup.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+#!/bin/bash
```
Comment (aleksandr-andreichenko-epam) [inline salt-minion/setup.sh:1]: :)

File: salt-minion/setup.sh [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+salt_minion_check () {
+	if [ ! -f /var/log/salt/minion ]; then
+  	  echo "File not found!" && \
+          touch /var/log/salt/minion && \
+	  salt_minion_check
+	else
+	  tail -f /var/log/salt/minion
+	fi
+}
```
Comment (Andreichenko) [inline salt-minion/setup.sh:11]: need to test

**Review by Andreichenko — COMMENTED** [review]
looks good

**Pull request merged**